test/fixtures/cookbooks/aws_test/recipes/ssm_parameter_store.rb

```diff
@@ -71,12 +109,15 @@
   owner 'ec2-user'
   group 'ec2-user'
   mode '0755'
-  sensitive true
+  sensitive false
```

Comment:
False is the default so you can just remove this

Reply:
changed it back to true.  I had changed it to false when I was testing.